### PR TITLE
Add yellow color to subtitle 3 using class

### PR DESCRIPTION
**Pull request by Blixt-A**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h3>Subtitle</h3>
             </p>
         </div>
         <div>
-            <h3>Subtitle</h3>
+            <h3 class="third-sub">Subtitle</h3>
```
Comment (idamakela) [inline index.html:37]: COMMENT!

**Review by idamakela — COMMENTED** [review]
see comment

**Comment by Blixt-A** [comment]
Hia

**Review by idamakela — APPROVED** [review]
NICE!

**Pull request merged**